Fitness/Controllers/HomeController.swift

```diff
@@ -317,63 +361,68 @@ extension HomeController: UICollectionViewDelegate, UICollectionViewDelegateFlow
 
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
         if collectionView.accessibilityIdentifier == Identifiers.todaysClassesCell {
-            // MARK: - Fabric
-            Answers.logCustomEvent(withName: "Found Info on Homepage", customAttributes: [
-                "Section": SectionType.todaysClasses.rawValue
-                ])
             let classDetailViewController = ClassDetailViewController()
             classDetailViewController.gymClassInstance = gymClassInstances[indexPath.row]
             navigationController?.hidesBottomBarWhenPushed = true
             navigationController?.pushViewController(classDetailViewController, animated: true)
             return
+        } else if collectionView.accessibilityIdentifier == Identifiers.allGymsCell {
```

Comment:
can we make a git issue to move collection view code like this to the file that contains the collection view?

Reply:
done!